src/controllers/author.js

```diff
@@ -0,0 +1,19 @@
+const { createItem, getAllItems, getItemById, updateItem, deleteItem } = require('../controllers/helpers')
+
+const createAuthor = (req, res) => createItem(res, 'author', req.body)
+
+const getAllAuthors = (req, res) => getAllItems(res, 'author')
+
+const getAuthorByID = (req, res) => getItemById(res, 'author', req.params.id)
+
+const updateAuthor = (req, res) => updateItem(res, 'author', req.body, req.params.id)
+
+const deleteAuthor = (req, res) => deleteItem(res, 'author', req.params.id)
```

Comment:
These could all be grouped together without making the content any less readable
```
const createAuthor = (req, res) => createItem(res, 'author', req.body)
const getAllAuthors = (req, res) => getAllItems(res, 'author')
const getAuthorByID = (req, res) => getItemById(res, 'author', req.params.id)
const updateAuthor = (req, res) => updateItem(res, 'author', req.body, req.params.id)
const deleteAuthor = (req, res) => deleteItem(res, 'author', req.params.id)
```